### PR TITLE
USHIFT-2506: CNCF remove parallel execution

### PR DESCRIPTION
**Pull request by eslutsky**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: test/scenarios-periodics/el92-src@cncf-conformance.sh [hunk 1]
```diff
@@ -41,8 +41,7 @@ run_sonobuoy() {
     ~/go/bin/sonobuoy run \
         --mode=certified-conformance \
         --dns-namespace=openshift-dns \
-        --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default \
-        --e2e-parallel=y 
+        --dns-pod-labels=dns.operator.openshift.io/daemonset-dns=default
```
Comment (ggiguash) [inline test/scenarios-periodics/el92-src@cncf-conformance.sh:44]: If we're merging this PR, please exclude the CNCF job from pre-submits as discussed on the Slack channel.

Comment (ggiguash) [reply test/scenarios-periodics/el92-src@cncf-conformance.sh:44]: /hold

**Comment by eslutsky** [comment]
depends on this PR https://github.com/openshift/microshift/pull/3115/

**Comment by eslutsky** [comment]
https://github.com/openshift/microshift/pull/3115 has been merged
/unhold

**Pull request merged**